src/index.ts

```diff
@@ -26,6 +26,13 @@ function createWindow() {
     return { action: 'deny' }
   })
 
+  win.webContents.on('will-navigate', (event, url) => {
```

Comment:
If the motivation is handling links in the same window, would the above `setWindowOpenHandler` solve this?

Reply:
that one is for links that don't open a new window (e.g. from PR comments)